examples/trials/network-morphism/config_pai.yml

```diff
@@ -0,0 +1,39 @@
+authorName: default
+experimentName: example_cifar10-network-morphism
+trialConcurrency: 1
+maxExecDuration: 24h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: pai
+#choice: true, false
+useAnnotation: true
```

Comment:
we don't have search space, so we don't need annotation. please set "false" here

Reply:
thank you for your kind reminder